Grigore Andrei Vlad/Senna/main.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{int n,i,aux,x;
+scanf("%d %d",&n,&x);
+for(i=1;i<=n;i++)
```

Comment:
Again, please start counting from 0, even though you might not be using an array here, trust me when I tell you, it will help

Reply:
Also, rather lazy solution, you're not even saving the numbers in an array first, but I guess no one said you *should* and it's not incorrect, so it's fine